dipy/align/mattes.pyx

```diff
@@ -1219,10 +1219,10 @@ cdef double _compute_mattes_mi(double[:, :] joint,
 
             if mi_gradient is not None:
                 for k in range(n):
-                    mi_gradient[k] -= joint_gradient[i, j, k] * factor
```

Comment:
That's the `-1 *` on line 258 of `immafine.py`? Just making sure that I understand this. Is it worth adding  comment here, so that we don't forget?